Empire/Empire.cpp

```diff
@@ -1387,6 +1387,18 @@ void Empire::AddBuildingType(const std::string& name) {
         return;
     m_available_building_types.insert(name);
     AddSitRepEntry(CreateBuildingTypeUnlockedSitRep(name));
+
+    // Decide whether to show a tutorial sitrep about building production.
```

Comment:
This decision is hard to follow; explaining the calculation and/or reason for it would be helpful.

Reply:
I'll revisit that once the question above about how to design the FOCS parts is resolved.

Reply:
This code was deleted in commit f4d47c23, so i think this conversation can be resolved.